sdp.js

```diff
@@ -551,6 +551,50 @@ SDPUtils.parseMsid = function(mediaSection) {
   }
 };
 
+// SCTP
+// parses either the older sctpmap format, or the newer sctp-port format
+SDPUtils.parseSctpDescription = function(mediaSection) {
+  var mline = SDPUtils.parseMLine(mediaSection);
+  var maxSizeLine = SDPUtils.matchPrefix(mediaSection, 'a=max-message-size:');
+  var maxMessageSize = maxSizeLine.length > 0
+    ? maxSizeLine[0].substr(19)
+    : '65536';
+  var sctpPort = SDPUtils.matchPrefix(mediaSection, 'a=sctp-port:');
+  if (sctpPort.length > 0) {
+    return {
+      number: sctpPort[0].substr(12),
+      protocol: mline.fmt,
+      maxMessageSize: maxMessageSize
+    };
+  }
+  var sctpMapLines = SDPUtils.matchPrefix(mediaSection, 'a=sctpmap:');
+  if (sctpMapLines.length > 0) {
+    var parts = SDPUtils.matchPrefix(mediaSection, 'a=sctpmap:')[0]
+      .substr(10)
+      .split(' ');
+    return {
+      number: parts[0],
+      protocol: parts[1],
+      maxMessageSize: maxMessageSize
+    };
+  }
+};
+
+// SCTP
+// outputs the draft-ietf-mmusic-sctp-sdp-26 version that all browsers
+// support by now receiving in this format
+SDPUtils.writeSctpDescription = function(media, sctp) {
+  var output = [
+    'm=' + media.kind + ' 9 ' + media.protocol + ' ' + sctp.protocol + '\r\n',
+    'c=IN IP4 0.0.0.0\r\n',
+    'a=sctp-port:' + sctp.number + '\r\n'
+  ];
+  if (sctp.maxMessageSize) {
```

Comment:
```suggestion
  if (sctp.maxMessageSize !== undefined) {
```

`0` is valid and means that there is no restriction.